### PR TITLE
Ability to configure whiteboard.

### PR DESCRIPTION
**Pull request by debendraoli**
- Ability to configure whiteboard.
- Replace deprecated es function `substr()` with `substring()`.

### CODE REVIEW

File: web/rootfs/defaults/settings-config.js [hunk 1]
```diff
@@ -63,7 +63,8 @@
 {{ $ENABLE_LOCAL_RECORDING_SELF_START := .Env.ENABLE_LOCAL_RECORDING_SELF_START | default "false" | toBool -}}
 {{ $DISABLE_PROFILE := .Env.DISABLE_PROFILE | default "false" | toBool -}}
 {{ $ROOM_PASSWORD_DIGITS := .Env.ROOM_PASSWORD_DIGITS | default "false" -}}
-
+{{ $WHITEBOARD_COLLAB_SERVER_BASE_URL := .Env.WHITEBOARD_COLLAB_SERVER_BASE_URL | default "" -}}
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:66]: Can you please rename this to be `WHITEBOARD_COLLAB_SERVER_PUBLIC_URL` ? That would make it analogous to Etherpad, where use the "base" URL when it's builtin, and the other one when it's external.

**Review by saghul — CHANGES_REQUESTED** [review]
The room_metadata plugin needs to be loaded too. We probably want to do that regardless of the whiteboard option, since it's generic.

**Review by saghul — APPROVED** [review]

**Comment by andys448** [comment]
Hi! 

ENV - WHITEBOARD_COLLAB_SERVER_BASE_URL not working correctly. After filling url - generated variable is still empty in web / config.js

**Comment by debendraoli** [comment]
correct env name is `WHITEBOARD_COLLAB_SERVER_PUBLIC_URL`.

**Comment by andys448** [comment]
> correct env name is `WHITEBOARD_COLLAB_SERVER_PUBLIC_URL`.

Thanks! Works fine)

**Comment by jvkassi** [comment]
Hello, the button whiteboard is only avaible for moderators. Is the default ? Can i change this config  ?

**Comment by jvkassi** [comment]
i was missing this config 
https://github.com/jitsi/docker-jitsi-meet/pull/1457

**Pull request merged**